src/runtime_src/core/tools/xbmgmt2/SubCmdConfigure.cpp

```diff
@@ -43,6 +43,31 @@ enum class config_type {
     reset
 };
 
+std::ostream&
+operator<<(std::ostream& os, const config_type& value)
+{
+  switch (value) {
+    case config_type::security:
+      os << "security";
+      break;
+    case config_type::clk_scaling:
+      os << "runtime clock scaling";
+      break;
+    case config_type::threshold_power_override:
+      os << "threshold power override";
+      break;
+    case config_type::threshold_temp_override:
+      os << "threshold temp override";
+      break;
+    case config_type::reset:
+      os << "clock scaling option reset";
+      break;
+    default:
```

Comment:
While I have approved the PR, this default case is not needed.  The compiler will detect a missing enumeration.